plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/FeatureDevSessionContext.kt

```diff
@@ -11,22 +11,22 @@ import com.intellij.openapi.vfs.VirtualFile
 import org.apache.commons.codec.digest.DigestUtils
 import software.aws.toolkits.core.utils.createTemporaryZipFile
 import software.aws.toolkits.core.utils.putNextEntry
-import software.aws.toolkits.jetbrains.services.amazonqFeatureDev.model.ZipCreationResult
 import java.io.File
 import java.io.FileInputStream
 import java.util.Base64
 import kotlin.io.path.Path
 import kotlin.io.path.relativeTo
 
 class FeatureDevSessionContext(val project: Project) {
+    // TODO: Need to correct this class location in the modules going further to support both amazonq and codescan.
 
-    private var _projectRoot = project.guessProjectDir() ?: error("Cannot guess base directory for project ${project.name}")
     private val ignorePatterns = listOf(
         "\\.aws-sam",
         "\\.svn",
         "\\.hg/",
         "\\.rvm",
         "\\.git/",
+        "\\.gitignore",
```

Comment:
issue: amazonqFeatureDev needs the .gitignore to be present in the zip file when uploading artifacts

Reply:
adding another method to avoid this.